roles/pulp_common/tasks/prepare_signing_service.yml

```diff
@@ -0,0 +1,80 @@
+---
```

Comment:
This is now duplicate code.

Could you perhaps remove this from the prepare.yml files and set a molecule variable instead for the molecule roles that need it?

Or check some variable that molecule always sets?

Reply:
We still need a variable so that this is not run for user systems.

Also, replace molecule/release-galaxy/prepare.yml with a symlink to ../release-static/prepare.yml and replace molecule/source-galaxy/prepare.yml with a symlink to ../source-static/prepare.yml . This was the implementation prior to 669dbdde710762c59a6840c1b97cca119d6f6c0a